website/components/features-list/feature.jsx

```diff
@@ -0,0 +1,44 @@
+import Button from '@hashicorp/react-button'
+import s from './style.module.css'
+
+export default function Feature({
+  number,
+  title,
+  subtitle,
+  infoSections,
+  cta,
+  image,
+}) {
+  return (
+    <div className={s.featureContainer}>
+      <div className={s.imageContainer}>
+        <img src={image} alt={title} />
+      </div>
+      <div className={s.featureTextContainer}>
+        <span className={s.listNumber}>{number}</span>
+        <div className={s.featureText}>
+          <h3 className={s.featureTitle}>{title}</h3>
+          <p className={s.featureSubtitle}>{subtitle}</p>
+          <div className={s.infoSection}>
+            {infoSections.map(({ heading, content }) => (
+              <div key={`${title}-${heading}`}>
```

Comment:
This title is unnecessary -- Keys in react are unique to the map -- so if there's another Feature that has a similar heading it won't cause an issue.

The `title` doesn't add any value here, as the title is the same value for each iteration of this map.

```suggestion
              <div key={heading}>
```